build.gradle

```diff
@@ -11,26 +10,50 @@ buildscript {
                 "com.github.ben-manes:gradle-versions-plugin:0.13.0",
                 "gradle.plugin.de.fuerstenau:BuildConfigPlugin:1.1.4",
                 "net.ltgt.gradle:gradle-apt-plugin:0.21",
-                "com.jfrog.bintray.gradle:gradle-bintray-plugin:1.7.3",
         )
     }
 }
 
 plugins {
     // Apply the java-library plugin to add support for Java Library
     id 'java-library'
-    id 'com.jfrog.bintray' version '1.8.4'
-    id "maven"
     id "maven-publish"
+    id "signing"
+    id "io.github.gradle-nexus.publish-plugin" version "1.1.0"
+}
+
+// Create variables with empty default values
+ext["ossrhUsername"] = ''
```

Comment:
These lines seem to be unnecessary since the default values are always overwritten with values from `System.getenv`. Can we drop them?